concourse/client/api.py

```diff
@@ -83,9 +82,21 @@ def _post(self, url: str, body: str="", headers={}):
     def _delete(self, url: str):
         return self.request_builder.delete(url)
 
-    @abstractmethod
-    def login(self, team: str, username: str, passwd: str):
-        raise NotImplementedError
+    def login(self, username: str, passwd: str):
```

Comment:
so this is the same for v4 and v5?
for the sake of focussing this PR's scope, could you mv this refactoring to a separate commit (may be pushed to master w/o review)